projects/torchscript/util.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
```

Comment:
let's move this whole thing into `parlai/torchscript` i guess, maybe `parlai/nn/torchscript` or something

Reply:
I don't have a preference - moved into `parlai/torchscript/` for now but happy to revisit